stores/local.go

```diff
@@ -399,17 +400,15 @@ func (st *Local) removeSector(ctx context.Context, sid abi.SectorID, typ SectorF
 }
 
 func (st *Local) MoveStorage(ctx context.Context, s abi.SectorID, spt abi.RegisteredProof, types SectorFileType) error {
-	dest, destIds, sdone, err := st.AcquireSector(ctx, s, spt, FTNone, types, false, AcquireMove)
+	dest, destIds, err := st.AcquireSector(ctx, s, spt, FTNone, types, false, AcquireMove)
 	if err != nil {
 		return xerrors.Errorf("acquire dest storage: %w", err)
 	}
-	defer sdone()
 
-	src, srcIds, ddone, err := st.AcquireSector(ctx, s, spt, types, FTNone, false, AcquireMove)
+	src, srcIds, err := st.AcquireSector(ctx, s, spt, types, FTNone, false, AcquireMove)
 	if err != nil {
 		return xerrors.Errorf("acquire src storage: %w", err)
 	}
-	defer ddone()
 
```

Comment:
```suggestion
st.localLk.Lock()
defer st.localLk.Unlock()
```

Otherwise race condition

Reply:
We can't race with anything here as we're not accessing anything in `st` that could change (`st.index` is global and has its own locks)

Reply:
But we transfer the sector (file) without any locks. Will other functions have not problems if we call them and the sector has not yet been moved?

Reply:
We hold per-sector locks on higher levels (in manager.go/FinalizeSector where this is called)